### ISSUE TITLE
Add option to disable color for text events or gray them out

### ISSUE THREAD

**Issue by madig** [hexchat/hexchat#1591]
I'm currently using HexChat because Polari errors out on some SSL connections. What struck me immediately is that chat windows are incredibly noisy due to all the color for events that I don't care about. I tried looking for a regexp that blots out all uses of %C in pevents.conf but gave up due to nesting.

I basically consider *everything* except user messages and accompanying user names noise (including timestamps) and want it out of my sight by e.g. graying it out while message text and user names remain black. Bonus points for collapsing consecutive joins/quits/status notifications.

**Comment by Arnavion** [comment]
`%C\d\d?(?:,\d\d?)?`

**Comment by madig** [comment]
Ah. I completely overlooked that I don't need to care about nesting.

In case anyone cares, I set the default text foreground color to some dark gray, set %C17 to black, removed all color codes from all events and put %C17 back in for channel messages and actions and my own messages. I also removed hostmasks from joins etc., shortened timestamps to %H:%M and disabled display of the previous channel buffer. The channel window is now satisfyingly clean.
